harness.sh

```diff
@@ -132,34 +107,75 @@ backuppass=${BACKUP_PASS}
 guidir=${GUI_DIR}
 EOF
 
-cat > "${NODES_ROOT}/dcrwctl.conf" <<EOF
+cat > "${NODES_ROOT}/dcrmwctl.conf" <<EOF
 rpcuser=${RPC_USER}
 rpcpass=${RPC_PASS}
 rpccert=${WALLET_RPC_CERT}
 rpcserver=127.0.0.1:19557
 EOF
 
-cat > "${NODES_ROOT}/wallet/wallet.conf" <<EOF
+cat > "${NODES_ROOT}/dcrvwctl.conf" <<EOF
+rpcuser=${RPC_USER}
+rpcpass=${RPC_PASS}
+rpccert=${WALLET_RPC_CERT}
+rpcserver=127.0.0.1:19562
+EOF
+
+cat > "${NODES_ROOT}/mwallet/mwallet.conf" <<EOF
+username=${RPC_USER}
+password=${RPC_PASS}
+rpccert=${WALLET_RPC_CERT}
+rpckey=${WALLET_RPC_KEY}
+logdir=${NODES_ROOT}/mwallet/log
+appdata=${NODES_ROOT}/mwallet
+simnet=1
+pass=${WALLET_PASS}
+EOF
+
+cat > "${NODES_ROOT}/vwallet/vwallet.conf" <<EOF
 username=${RPC_USER}
 password=${RPC_PASS}
 rpccert=${WALLET_RPC_CERT}
 rpckey=${WALLET_RPC_KEY}
-logdir=./log
-appdata=./data
+logdir=${NODES_ROOT}/vwallet/log
+appdata=${NODES_ROOT}/vwallet
 simnet=1
+enablevoting=1
+enableticketbuyer=1
+ticketbuyer.limit=2
```

Comment:
```
~/harness/vwallet$ dcrwallet -C vwallet.conf --create                             
vwallet.conf:10: unknown option: ticketbuyer.limit 
```

`limit` is not a valid `ticketbuyer` option.

Are you looking for `ticketbuyer.balancetomaintainabsolute`?